opencti-platform/opencti-front/src/private/components/observations/indicators/IndicatorDetails.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { FunctionComponent } from 'react';
+import { createFragmentContainer, graphql } from 'react-relay';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import { SettingsApplications } from '@mui/icons-material';
+import ListItemText from '@mui/material/ListItemText';
+import Chip from '@mui/material/Chip';
+import Divider from '@mui/material/Divider';
+import makeStyles from '@mui/styles/makeStyles';
+import { IndicatorDetails_indicator$data } from '@components/observations/indicators/__generated__/IndicatorDetails_indicator.graphql';
+import ItemScore from '../../../../components/ItemScore';
+import IndicatorObservables from './IndicatorObservables';
+import ExpandableMarkdown from '../../../../components/ExpandableMarkdown';
+import ExpandablePre from '../../../../components/ExpandablePre';
+import ItemBoolean from '../../../../components/ItemBoolean';
+import StixCoreObjectKillChainPhasesView from '../../common/stix_core_objects/StixCoreObjectKillChainPhasesView';
+import { useFormatter } from '../../../../components/i18n';
+// eslint-disable-next-line import/extensions -- No idea...
+import { Theme } from '../../../../components/Theme';
```

Comment:
should be import type. And there shouldn't be // eslint-disable-next-line

Reply:
the error was "missing file extension" it's quite misleading. You're right using type fixes it.